gridmap/job.py

```diff
@@ -933,9 +936,52 @@ def grid_map(f, args_list, cleanup=True, mem_free="1G", name='gridmap_job',
             for job_num, args in enumerate(args_list)]
 
     # process jobs
-    job_results = process_jobs(jobs, temp_dir=temp_dir, white_list=white_list,
+    job_results = process_jobs(jobs, temp_dir=temp_dir,
+                               white_list=white_list,
                                quiet=quiet, local=local,
                                max_processes=max_processes)
 
+    # send a completion mail (if requested and configured)
+    if completion_mail and SEND_ERROR_MAIL:
+        send_completion_mail(name=name)
+
     return job_results
 
+
+def send_completion_mail(name):
```

Comment:
Since this shares a lot of code with `send_error_mail`, I'd much prefer to see some light refactoring so that there was a `_send_mail` function that would take a subject, body text, and optional attachments that both `send_error_mail` and `send_completion_mail` used under the hood.  This would cut down on the code duplication a bit.